### DISCUSSION TITLE
error with github pages. help

### DISCUSSION THREAD

**Discussion by BakSnn** [community/community#108603] [Actions]
### Select Topic Area

Bug

### Body

I want to deploy react applications (website) via github pages, I enter the npm run deploy command, but nothing happens, an endless build, and then the deploy is canceled. What to do?
![2](https://github.com/community/community/assets/112386046/07d78c00-5868-4e96-a087-7f93acc9645e)

**Comment by Cizr** [comment]
start by checking the terminal and browser console for any error messages. Ensure that your package.json file contains the correct deployment scripts, and confirm the existence of the gh-pages branch.
Verify that the repository URL in package.json under "homepage" is accurately set with your GitHub username and repository name. Before deploying, clean up previous build artifacts using npm run clean or by manually deleting the build directory. Keep npm packages up-to-date with npm install. (if it kept happening share specific details or error messages encountered during deployment )

**Reply by BakSnn** [reply]
I did everything you said, but nothing has changed. Just an endless build and within an hour it breaks
![3](https://github.com/community/community/assets/112386046/ae356a15-3c0d-4060-9712-2bf02275b53e)

**Reply by BakSnn** [reply]
there are no errors in the console
![4](https://github.com/community/community/assets/112386046/cea5bdd8-d2d2-4727-9796-8702f3664269)

**Reply by BakSnn** [reply]
what to do?

**Reply by AdriSel25** [reply]
"Did you manage to figure out the solution to the problem? I'm struggling with it too."
